package.json

```diff
@@ -8,7 +8,8 @@
     "migrations": "node db/migrations.js",
     "seeds": "node db/seeds.js",
     "start": "node app.js",
-    "test": "nyc --reporter=text mocha --recursive --exit"
+    "test": "nyc --reporter=text mocha --recursive --exit",
+    "test-watch": "mocha --recursive --watch"
```

Comment:
I like to have test-watch so I can run tests continually while making changes, helps me a lot with TDD